src/app.rs

```diff
@@ -187,6 +190,9 @@ impl App {
             "To get started, take a look at the configuration file, which is located in ~/.config/termviz/termviz.yml.",
             "",
             "Press any key to go back to the robot view, or Ctrl+c to exit.",
+            "Enter 't' to to enter/exit teleoperating mode where the configured keys are used move the robot. \
```

Comment:
put this before the new line:
"to get started blahblah",
"Enter 't' blah blah",
"",
"Press any blah"
"",

Reply:
Also:
- "Press 't'
- to to (typo)
- command vel (typo) or maybe directly write cmd_vel

Reply:
Done, however I had the newline first, but it looks better like this

Reply:
I don't see any change!

Reply:
Hmm its changed, no idea why its not showing up